lib/topological_inventory/openshift/operations/worker.rb

```diff
@@ -60,24 +60,78 @@ def order_service(payload)
           )
           logger.info("Ordering #{service_offering.name} #{service_plan.name}...Complete")
 
+          context = svc_instance_context_with_url(service_offering, service_plan, service_instance )
+          status  = provisioning_status(service_instance)
+
+          update_task(task_id, :state => "completed", :status => status, :context => context)
+        rescue StandardError => err
+          update_task(task_id, :state => "completed", :status => "error", :context => {:error => err.to_s})
```

Comment:
FYI we just added some extra logging which is why this is conflicting now

Reply:
I see, thanks, rebasing